### ISSUE TITLE
material-ui-icons deprecated in favour of @material-ui/icons

### ISSUE THREAD

**Issue by geraldodev** [InsidersByte/bs-material-ui-icons#3]
Can you update to beta.42 ?

Is this code generated by a script ?

For the moment I've created a MaterialUIIcons.re on the project and pasted/changed the declaration of the icon I need to use
```
module Menu = {
  [@bs.module "@material-ui/icons/Menu"] external reactClass : ReasonReact.reactClass = "default";
  let make = (children) =>
    ReasonReact.wrapJsForReason(~reactClass, ~props=Js.Obj.empty(), children);
};
```

**Comment by InsidersByte** [comment]
Hey @geraldodev,

I see that they deprecated [material-ui-icons](https://www.npmjs.com/package/material-ui-icons) in favour of  [@material-ui/icons](https://www.npmjs.com/package/@material-ui/icons)

I will update this project to use the new package in the next couple of days

**Comment by geraldodev** [comment]
Thank you for your efforts on wrapping material-ui.

Very much appreciated.

**Comment by InsidersByte** [comment]
@geraldodev this has been done and released in version `0.2.0`
